src/ITensorChainRules/ITensorChainRules.jl

```diff
@@ -1,27 +1,181 @@
-@reexport module ITensorChainRules
+module ITensorChainRules
 
 using ChainRulesCore
 using ITensors
 
+include("zygoterules.jl")
+
 function ChainRulesCore.rrule(::typeof(getindex), x::ITensor, I...)
   y = getindex(x, I...)
   function getindex_pullback(ȳ)
-    x̄ = ITensor(inds(x))
+    # TODO: add definition `ITensor(::Tuple{}) = ITensor()`
+    # to ITensors.jl so no splatting is needed here.
+    x̄ = ITensor(inds(x)...)
```

Comment:
Does it make sense to also set the I' to be NoTangent()? I think we generally don't want to define diff for the index

Reply:
That is what I do below in the line `Ī = broadcast(_ -> NoTangent(), I)` or else maybe I am misunderstanding your question.

Reply:
I see, I thought line 15 is doing something else. Never mind